ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Operation/09_SplitBooking.md

[!!User Interface split booking](../UserInterface/01a_Bookings.md#split-booking)   
[!!User Interface book](../UserInterface/01_Book.md)  

# Split a booking

In some occasions, it may be necessary to split a booking to keep track of different items separately, for example if an invoice includes products with different tax rates or if total costs must be divided into two or more cost centres. Split bookings can affect both debit and credit accounts, that is, incoming and outgoing invoices.

> 
> The [SPLIT] button in the *BOOKINGS* tab allows to split bookings during the manual booking process. For detailed information about creating a manual booking, see [Create a manual booking](./04_CreateManualBooking.md).
> 

Reviewer: ```suggestion
You can split postings during the manual posting process. For detailed information about creating a manual posting, see [Create a manual posting](./04_CreateManualBooking.md).
```
Geht das nur für manuelle Buchungen?

> Once the splitting has been performed, the different split (partial) bookings are assigned to the corresponding accounts. The split bookings are then displayed in the bookings list underneath the main booking.

Reviewer: ```suggestion
Once the splitting has been performed, the different split (partial) postings are assigned to the corresponding accounts. The split postings are displayed in the postings list below the main posting.
```

> > [Info] Splitting a booking must be performed during the manual booking process. When splitting an invoice after the booking has been automatically registered following a business transaction or after it has been manually created, the system creates a copy of the existing main booking with the newly assigned contra accounts.

Reviewer: ```suggestion
```

#### Prerequisites

> - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
> - A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).
> - A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).

Reviewer: ```suggestion
- A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).
- At least one posting has been created, see [Create a manual booking](./04_CreateManualBooking.md).
```

#### Procedure

*Accounting > Select fiscal year > Book > Tab BOOKINGS*

![Split booking](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/SplitBooking.png "[Split booking]")

> 1. Select the appropriate currency for the booking in the *Currency code* drop-down list.

Reviewer: ```suggestion
1. Click the *Currency code* drop-down list and select the appropriate currency for the posting. All available currencies are displayed in the list.
```

> 1. Select the appropriate currency for the booking in the *Currency code* drop-down list.

Reviewer: ```suggestion
1. If necessary, click the *Currency code* drop-down list and select the appropriate currency for the posting. All available currencies are displayed in the list. By default, the base currency is preselected.
```

  > [Info] The booking numbers must be sequential and are added by the system automatically.

2. Enter the total invoice value in the *Turnover* field.

3. Enter the date for the manual booking in the *Date* field. Date format must be DDMMYY.

4. Enter a personal account number in the *Account* field, that is, a debtor (customer) or a creditor (provider) account number.

  > [Info] The fields *Date* and *Account* must be filled out to be able to split a booking. The *Contra Account* field can be specified when splitting the booking.

5. Enter any other relevant booking information in the corresponding fields.

>   > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent bookings taking this field as reference.

Reviewer: vgl. manual booking

6. Click the [SPLIT] button.  
The *Split booking* window is displayed.  

  ![Split booking window](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/SplitBooking01.png "[Split booking window]")

> 7. Split the booking as necessary assigning each split booking to the corresponding contra account.

Reviewer: ```suggestion
7. Split the posting in the *Split posting* window by following the steps below:
  + Enter the partial amount for the split posting in the *Turnover* field.
  + Enter the corresponding contra account for the split posting in the *Contra account* field.
  + Click the [POST] button.    
  The split posting is displayed in the *Split posting* window below the main posting.
```

> 8. Click the [BOOK] button after each split booking.  

Reviewer: ```suggestion
8. Repeat the step **7** for each split posting.
```

A message in the top left corner of the *Split booking* window confirms each split booking. The split bookings are now listed underneath the main booking in the *Split booking* window.

  ![Bookings split](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/SplitBooking02.png "[Bookings split]")

>   > [Info] When the total invoice amount has been divided, the remainder will equal 0. At this point, the [SAVE] button is activated and can be selected.

Reviewer: ```suggestion
  > [Info] As soon as the total posting amount in the *Split posting* window equals *0.00*, the [SAVE] button is unlocked.
```

9. Click the [SAVE] button.  

> A message in the top left corner of the *Split booking* window confirms that the split bookings have been recorded.

Reviewer: ```suggestion
The notice *Split posting applied* is displayed in the top left corner of the *Split booking* window. The *Split booking* window is closed automatically. The split posting is displayed in the postings list below the corresponding main posting.
```

#### Prerequisites

A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).

#### Procedure

*Accounting > Post > Tab POSTINGS*

![Postings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/Bookings.png "[Postings]")

1. If necessary, click the *Currency code* drop-down list and select the appropriate currency for the posting. All available currencies are displayed in the list. By default, the base currency is preselected.

2. Enter the posting amount in the *Turnover* field. This field is mandatory.  

  > [Info] The fields *Turnover*, *Date* and *Account* must be filled to split a posting. The *Contra Account* field can be specified when splitting the posting.

3. Enter a reference number, usually the invoice number, in the *Receipt 1* field.

  > [Info] Even though it is not mandatory, it is highly recommended to fill in the *Receipt 1* field, as it is used as a reference to identify and assign any subsequent postings, for example for an invoice settlement or a credit note in case of refund.

> 

Reviewer 2: see comment in chapter 04_Create manual booking
